app/assets/stylesheets/application.scss

```diff
@@ -1,25 +1,22 @@
 // The main application stylesheet
 
-// govuk_frontend_toolkit
+// Settings
+// Mixins and varibles from the govuk_frontend_toolkit
 @import "colours";
 @import "measurements";
 @import "shims";
 @import "typography";
 
-// Mixins
-@import "mixins/helpers";
-@import "mixins/margins";
+// Components
+@import "components/change-history";
+@import "components/metadata";
+@import "components/notice";
+@import "components/page-contents";
+@import "components/page-header";
+@import "components/related-content";
+@import "components/service-standard-point";
+@import "components/subsections";
```

Comment:
Hadn't considered that, good point.

We could use `/service-manual-components/` to make it clear they are components for this project.